Microsoft.Dynamics365.UIAutomation.Sample/UCI/BusinessProcessFlow/BusinessProcessFlowNextStage.cs

```diff
@@ -14,7 +14,7 @@ public class BusinessProcessFlowNextStage : TestsBase
         [TestCleanup]
         public override void FinishTest() => base.FinishTest();
 
-        public override void NavigateToHomePage() => _xrmApp.Navigation.OpenSubArea("Sales", "Leads");
+        public override void NavigateToHomePage() => NavigateTo(UCIAppName.Sales, "Sales", "Leads");
```

Comment:
I like the idea of breaking this out but would really like the samples to be simple.  I've noticed that Testers that aren't developers are a little confused by even "UCIAppName.Sales"  like all they have to do is change the ".Sales" to their custom app name and it will work.    We may revert that back to a string value just to not cause confusion with non developers getting started.

Reply:
Would you be interested in doing a conference call to discuss the changes?

Reply:
Thanks @dtu, For me are String values OK but that was before also a constant just in two lines,
```csharp
                xrmApp.Navigation.OpenApp(UCIAppName.Sales);
                xrmApp.Navigation.OpenSubArea("Sales", "Leads");
```
Hier is just being doing once for all tests in this class. But the old versions still working.
Maybe is NavigateToHomePage not the right name.

We can make a conference call. No problem.

Reply:
Please let me know, if this is the only blocker I will remove the comment to keep moving forward.
Thanks.